REQUIRE

```diff
@@ -1,11 +1,11 @@
 julia 0.7
 Automa
-BioCore
-BioSymbols 2.0
+BioCore 2.0.0
+BioSymbols 3.0.1
 BufferedStreams
 Combinatorics
 IndexableBitVectors
-IterTools 0.1
 IntervalTrees
 Twiddle
 DataStructures
+Nullables
```

Comment:
It is okay to have this now but I think that in general we should not depend on deprecated/removed features. We may need a second round to review our APIs.